draft-ietf-acme-acme.md

```diff
@@ -318,6 +318,17 @@ serialization, with the protected header and payload expressed as
 base64url(content) instead of the actual base64-encoded value, so that the content
 is readable.  Some fields are omitted for brevity, marked with "...".
 
+## Equivalence of JWKs
+
+At several points in the protocol, it is necessary for the server to determine
+whether two JWK objects represent the same key.  In performing these checks, the
+server SHOULD consider two JWKs to match if and only if they have the same "kty"
+value and contain identical values in the fields used in the computation of a
+JWK thumbprint for that key type:
+
+* "RSA": "n", "e"
+* "EC": "crv", "x", "y"
+
```

Comment:
JSHA objected to the additional fingerprint computations.  I will rephrase, though, to make clear that this is equivalent to comparing fingerprints, modulo hash collisions.